.travis.yml

```diff
@@ -1,7 +1,7 @@
 sudo: false
 language: node_js
 node_js:
-  - "6.2"
+  - "6.10.3"
```

Comment:
I am really not in the mood for updating this every time Node releases a minor version.